app/controllers/admin/events_controller.rb

```diff
@@ -52,6 +52,7 @@ def show
       @versions = @event.versions |
        PaperTrail::Version.where(item_type: 'Commercial').where_object(commercialable_id: @event.id, commercialable_type: 'Event') |
        PaperTrail::Version.where(item_type: 'Commercial').where_object_changes(commercialable_id: @event.id, commercialable_type: 'Event')
+      add_vote_versions_to_versions
```

Comment:
Sorry for late response but I tried that approach it did not work out. Since it creates a like query on object_changes attribute some vote versions of a particular event were shown in other events too.